docs/extend/extend-desktop/mvd-buildingplugins.md

```diff
@@ -21,7 +23,7 @@ Before you can build a plugin you must install all [prerequisites](https://githu
     - Run the `npm run start` command to compile in real-time. Until you stop the script, it compiles code changes as you make them.
 
 ## Building server content
-1. Navigate to the plugin directory. If there is no `/nodeServer` directory in the plugin directory, procede to the **Building Javascript content (*.js files)** section.
+1. Navigate to the plugin directory. If there is no `/nodeServer` directory in the plugin directory, procede to the **Building Javascript content (*.js files)** section below.
```

Comment:
Note that server content can be in the form of a  nodejs JavaScript router or a c-based ZSS plugin. We're missing a section for "Building ZSS content". I think the as-is "Building Server Content" and "Building Javascript Content" should be repurposed into a "Building Javascript services" and that a "Building ZSS services" should be started by referencing the example here: https://github.com/jordanfilteau1995/hello-world-data-service